lms/templates/fields/field_readonly.underscore

```diff
@@ -1,10 +1,8 @@
 <label class="u-field-title" aria-hidden="true">
```

Comment:
@cahrens this label is hidden from screen readers but is still technically visible in the DOM as an empty element.  I could trip up those who use voice input, and will definitely get picked up by automated tools (because ```<label>``` elements are only valid if they have an associated form field).  It would be great if we can make this display:none; like the other labels on the page.